Meta/Lagom/Tools/CodeGenerators/LibTimeZone/GenerateTimeZoneData.cpp

```diff
@@ -530,17 +530,17 @@ struct TimeZoneOffset {
 };
 
 struct DaylightSavingsOffset {
-    AK::Time time_in_effect(AK::Time time) const
+    AK::UnixDateTime time_in_effect(AK::UnixDateTime time) const
     {
         auto in_effect = this->in_effect;
-        in_effect.year = seconds_since_epoch_to_year(time.to_seconds());
+        in_effect.year = seconds_since_epoch_to_year(time.seconds_since_epoch());
 
         return in_effect.time_since_epoch();
     }
 
     i64 offset { 0 };
-    AK::Time year_from {};
-    AK::Time year_to {};
+    AK::UnixDateTime year_from {};
+    AK::UnixDateTime year_to {};
```

Comment:
(looking just at names, iʼd say that integers could also be a thing; not sure if it would make any code simpler, though)

Reply:
These were dates previously, as they make calculations easier later on. In fact, there may be sub-year precise DST changes.